site/README.md

```diff
@@ -73,25 +73,19 @@ When running in development environments, avatars go to the `dev/avatars/(user.i
 
 ### Serving Blocks
 
-Before serving any blocks, they need to be built. Blocks can be registered in the repo's `/hub`
-with a build-config. The build-script `/site/scripts/build-blocks.sh` allows to build blocks
-individually. It requires the commandline tool:
-
-- `curl`
-- `jq`
-- `md5sha1sum`
-- `rsync`
-
-These can be installed by your cli pkg mngr of choice (use `brew` on macOS).
+Before serving any blocks, they need to be prepared (i.e. built in most cases).
+Blocks can be registered in the repo's `/hub` with a build-config.
+The build-script `yarn exe yarn exe site/scripts/prepare-blocks.ts` allows to prepare blocks.
 
 ```sh
-# build one or more blocks
-yarn build-block ./hub/@hash/paragraph.json
-# build all blocks
-yarn build-blocks
+# prepare all blocks
+yarn exe yarn exe site/scripts/prepare-blocks.ts
+
+# prepare one or more blocks
```

Comment:
may match none
```suggestion
# prepare blocks matching a filter (in this example, any in the `hub/@hash` folder)
```

Reply:
Addressed in 74d69756f058ddd27683891898df597c8cbd9268